Readme.md

```diff
@@ -0,0 +1,3 @@
+# Welcome to Nextcloud! :iphone: :cloud: :computer:
```

Comment:
We should insert the actual emoji characters here as text does not support the `:` syntax when opening